.gitignore

```diff
@@ -30,4 +30,6 @@ bin/*
 go.work
 
 config/crd/bases/_.yaml
-.idea
\ No newline at end of file
+.idea
+
+config/crd/external/
```

Comment:
Why do we need to add this?

Reply:
While working on unit tests I had a panic that crashed the tests and cleanup did not happen. This is just in case it were to happen again so we do not risk to commit unnecessary files. It's just for convenience.